manifests/m/Mobirise/Mobirise/5.3.10/Mobirise.Mobirise.installer.yaml

```diff
@@ -1,10 +1,20 @@
-# Automatically updated by the winget bot at 2021/Jun/10
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
 PackageIdentifier: Mobirise.Mobirise
 PackageVersion: 5.3.10
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
 Installers:
-- Architecture: neutral
+- InstallerLocale: en-US
+  Architecture: x86
   InstallerType: nullsoft
+  Scope: machine
```

Comment:
```suggestion
  Scope: user
```
quick fix 😬

Reply:
![folder](https://user-images.githubusercontent.com/69106310/126062527-a62e71d1-e5b8-431c-b035-c039e7c10674.png)

This is the install location...
@OfficialEsco